site/en/docs/webstore/i18n/index.md

```diff
@@ -157,134 +149,149 @@ app that has English as its default language and British English as its secondar
 }
 ```
 
-If you add a locale later, you just need to add a `_locales/_locale_/messages.json` file for that
+If you add a locale later, you just need to add a `_locales/LOCALE_CODE/messages.json` file for that
 new locale. For example, if you add support for French and Arabic, you'll need to add two files:
 `_locales/fr/messages.json` and `_locales/ar/messages.json`. Each of those files can be a copy of
-`_locales/en_GB/messages.json`, but with translated versions of the "message" values.
+`_locales/en_GB/messages.json`, but with translated versions of the `message` values.
 
-For more details, see the following examples and documentation.
+For more details, see the extension [Internationalization][i18n] documentation: 
+    - The [How extensions find strings][searching-messages] section shows how you can sometimes omit message strings.
+    For example, if the extension's name isn't translated, you can leave it out for non-default locales.
+    - The [How to set your browser's locale][set-browser-locale] section is helpful when you test your translations.
```

Comment:
```suggestion

- The [How extensions find strings][searching-messages] section shows how you can sometimes omit
  message strings. For example, if the extension's name isn't translated, you can leave it out for
  non-default locales.
- The [How to set your browser's locale][set-browser-locale] section is helpful when you test your
  translations.
```

The preview shows that this content is all being rendered as one paragraph. This suggestion reformats the text so Markdown should interpret it as a bulleted list.